src/service/main.cpp

```diff
@@ -20,6 +20,7 @@
 #include <Picoscope4000a.hpp>
 #endif
 
+#include "settings.h"
```

Comment:
```suggestion
#include <settings.h>
```
includes a different opendigitizer cmake module -> not a relative path and the corresponding path gets added to the search path for bracket include.

Reply:
This is actually a more complicated case, wherein the /utils is not compiled in ui-wasm, because that gets build in from /ui.
I added ../utils as common_utils for ui-wasm builds.

Reply:
ah yes, I'll need to revisit that cmake setup, having shared modules between the UI and service part was not a requirement when I set this up. OK for this PR.